content/v2.0/influxdb-templates/create.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Create an InfluxDB template
+description: >
+  Use the InfluxDB UI and the `influx pkg export` command to create InfluxDB templates.
+menu:
+  v2_0:
+    parent: InfluxDB templates
+    name: Create a template
+    identifier: Create an InfluxDB template
+weight: 102
+v2.0/tags: [templates]
+---
+
+Use the InfluxDB user interface (UI) and the `influx pkg export` command to
+create InfluxDB templates.
+Add resources (buckets, Telegraf configurations, tasks, and more) in the InfluxDB
+UI and export the resources as a template.
+
+{{% note %}}
+Templatable resources are scoped to a single organization, so the simplest way to create a
+template is to create a new organization, build the template within the organization,
+and then [export all resources](#export-all-resources) as a template.
+
+**InfluxDB OSS** supports multiple organizations so you can create new organizations
+for the sole purpose of building and maintaining a template.
+In **InfluxDB Cloud**, your user account is an organization.
+**We recommend using InfluxDB OSS to create InfluxDB templates.**
+{{% /note %}}
+
+**To create a template:**
+
+1. [Start InfluxDB](/v2.0/get-started/).
+2. [Create a new organization](/v2.0/organizations/create-org/).
+3. In the InfluxDB UI add one or more of the following templatable resources:
+
+   - [buckets](/v2.0/organizations/buckets/create-bucket/)
+   - [checks](/v2.0/monitor-alert/checks/create/)
+   - [dashboards](/v2.0/visualize-data/dashboards/create-dashboard/)
+   - [dashboard variables](/v2.0/visualize-data/variables/create-variable/)
+   - [labels](/v2.0/visualize-data/labels/)
+   - [notification endpoints](/v2.0/monitor-alert/notification-endpoints/create/)
+   - [notification rules](/v2.0/monitor-alert/notification-rules/create/)
+   - [tasks](/v2.0/process-data/manage-tasks/create-task/)
+   - [Telegraf configurations](/v2.0/write-data/use-telegraf/)
+
+4. Export the template _(see [below](#export-a-template))_.
+
+{{% warn %}}
+InfluxDB templates do not support the [table visualization type](/v2.0/visualize-data/visualization-types/table/).
+Dashboard cells that use table visualization are not included in exported templates.
+{{% /warn %}}
+
+## Export a template
+Do one of the following to export a template:
+
+1. Export all resources in an organization _(recommended)_
+2. Export specific resources in an organization
+
+### Export all resources
+To export all templatable resources within an organization to a template manifest,
+use the `influx pkg export all` command.
+Provide the following:
+
+- **Organization name** or **ID**
+- **Authentication token** with read access to the organization
+- **Destination path and filename** for the template manifest.
+  The filename extension determines the template format—both **YAML** (`.yml`) and
+  **JSON** (`.json`) are supported.
+
+###### Export all resources to a template
+```sh
+# Syntax
+influx pkg export all -o <org-name> -f <file-path> -t <token>
```

Comment:
should `-t` token be here? we don't list other global params like `--host`

Reply:
I included in the command since it's called out specifically above. I can remove it from the example.